Test/Test.csproj

```diff
@@ -1,80 +1,13 @@
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{DD1A3698-ED72-49C8-874F-F721EA8012B6}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Test</RootNamespace>
-    <AssemblyName>Test</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
+    <TargetFrameworks>net45;netcoreapp3.0</TargetFrameworks>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
-    <DebugSymbols>true</DebugSymbols>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <DebugType>full</DebugType>
-    <PlatformTarget>AnyCPU</PlatformTarget>
-    <ErrorReport>prompt</ErrorReport>
-    <Prefer32Bit>false</Prefer32Bit>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <Optimize>true</Optimize>
-    <DebugType>pdbonly</DebugType>
-    <PlatformTarget>AnyCPU</PlatformTarget>
-    <ErrorReport>prompt</ErrorReport>
-    <Prefer32Bit>false</Prefer32Bit>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="Mono.Options, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Mono.Options.4.4.0.0\lib\net4-client\Mono.Options.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="protobuf-net, Version=2.0.0.668, Culture=neutral, PublicKeyToken=257b51d87d2e4d67, processorArchitecture=MSIL">
-      <HintPath>..\packages\protobuf-net.2.0.0.668\lib\net40\protobuf-net.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.XML" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="CustomSerializers.cs" />
-    <Compile Include="ITest.cs" />
-    <Compile Include="ITestSpec.cs" />
-    <Compile Include="MessageTestSpec.cs" />
-    <Compile Include="MemStreamTest.cs" />
-    <Compile Include="Messages.cs" />
-    <Compile Include="NetTest.cs" />
-    <Compile Include="Program.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="MyRandom.cs" />
-    <Compile Include="SerializerSpecimen.cs" />
-    <Compile Include="Tester.cs" />
-    <Compile Include="MessageTest.cs" />
-  </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\NetSerializer\NetSerializer.csproj">
-      <Project>{85A11D07-8D18-42D5-ACCF-EF9744EFE825}</Project>
-      <Name>NetSerializer</Name>
-    </ProjectReference>
+    <PackageReference Include="protobuf-net" Version="3.0.0-alpha.43" />
+    <PackageReference Include="Mono.Options" Version="5.3.0.1" />
```

Comment:
Update the package dependencies separately. They shouldn't be part of this patch.

Reply:
Yeah the problem is that both of the packages were too old to run on .NET Core so I had to update them. I'll make a separate PR though.

Reply:
No need for a separate PR, just have the change in a separate commit.